Automattic/wp-calypso
# Stats: Update promo card links to Jetpack mobile app

#### Proposed Changes

The current links and QR code (as pulled from the Figma design doc) direct us to the WordPress mobile app. I've updated the URLs and QR code to instead direct us to the Jetpack mobile app.

#### Testing Instructions



* Check out this branch locally.
* Spin up Storybook via `yarn workspace @automattic/components run storybook`.
* Check the Promo Card stories. Ensure the links go to the Jetpack mobile app page.
* Scan the QR code and confirm it goes to the same page.

Updated image showing minor changes.

![JetpackMobilePromoCard](https://user-images.githubusercontent.com/40267301/201880529-83b9166b-38d9-4507-af1a-e72f58d06c45.png)

#### Pre-merge Checklist





Related to #69230 

Depends on #70023
